udata/core/dataset/api_fields.py

```diff
@@ -101,6 +101,14 @@
 community_resource_page_fields = api.model(
     'CommunityResourcePage', fields.pager(community_resource_fields))
 
+#: Default mask to make it lightweight by default
+DEFAULT_MASK = ','.join((
```

Comment:
It seconds my thoughts on consistency for masks, we should stick to lists on the js side too.